etc/portage/repo.postsync.d/sync_gentoo_cache

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+source /etc/init.d/functions.sh
+source /etc/portage/util-functions.sh
+
+repository_name=${1}
+repository_path=${3}
+
+# Number of jobs for egencache, default is number or processors.
+parallel_jobs="$(nproc)"
+
+if [ "${repository_name}" == "gentoo" ]; then
+	ebegin "Fetching pre-generated metadata cache for ${repository_name}"
+	rsync -aq rsync://rsync.gentoo.org/gentoo-portage/metadata/md5-cache/ "${repository_path}"/metadata/md5-cache/
+	eend $?
+
+	ebegin "Updating metadata cache for ${repository_name}"
```

Comment:
Yes, that's what I did initially, but I think it does not make sense. Please read my justification in the third paragraph above. Of course, I can still move it outside if want.